parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -420,9 +426,21 @@ extractKubeBinaries() {
         k8s_tgz_tmp="${k8s_downloads_dir}/${k8s_tgz_tmp_filename}"
         mkdir -p ${k8s_downloads_dir}
 
-        retrycmd_get_tarball 120 5 "${k8s_tgz_tmp}" ${kube_binary_url} || exit $ERR_K8S_DOWNLOAD_TIMEOUT
-        if [[ ! -f ${k8s_tgz_tmp} ]]; then
-            exit "$ERR_K8S_DOWNLOAD_TIMEOUT"
+        registry_regex='^.+\/.+\/.+:.+$'
```

Comment:
Can we add a comment about exactly what this regex is doing for future clarity

Reply:
comment added in the previous line for now
`# if the url is a registry url, use oras to pull the artifact instead of curl`